workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectLinker.cs

```diff
@@ -13,6 +13,7 @@ public class EntityGameObjectLinker
         private readonly WorkerSystem worker;
         private readonly EntityManager entityManager;
         private readonly HashSet<Type> gameObjectComponentTypes = new HashSet<Type>();
+        private const long WorkerEntityId = 0;
```

Comment:
Why not make this the `EntityId` directly?

Reply:
Because the compiler won't let that happen with `const` so I had to choose between `const long` or `readonly EntityId` and chose the former. Would `readonly EntityId` be better?

Reply:
static readonly 👍

Reply:
Will change to `static readonly EntityId`